_config.yml

```diff
@@ -1,12 +1,14 @@
-plugins:
-  - jekyll-relative-links
-relative_links:
-  enabled: true
-  collections: true
-include:
-  - API.md
-  - README.md
-  - Specification.md
-  - LICENSE
```

Comment:
this is also unnecessary, all files are included by default unless the file name starts with `.` or `_` or a few other special chars